VagrantProvision.sh

```diff
@@ -7,14 +7,17 @@ source ~/.profile
 
 #To get rid of "dpkg-preconfigure: unable to re-open stdin: No such file or directory" warnings
 export DEBIAN_FRONTEND=noninteractive
+if [ -f /etc/apt/apt.conf.d/70debconf ]; then
+  sudo sed -i 's/ --apt//' /etc/apt/apt.conf.d/70debconf
+fi
 
 echo "Updating OS..."
 sudo apt-get -qq update > Ubuntu_upgrade.txt 2>&1
 sudo apt-get -q -y upgrade >> Ubuntu_upgrade.txt 2>&1
 sudo apt-get -q -y dist-upgrade >> Ubuntu_upgrade.txt 2>&1
 
 echo "### Setup NTP..."
-sudo apt-get -q -y install ntp
+sudo apt-get -q -y install ntp 2>&1
```

Comment:
Yeah, it does.  It seems that quite a bit of the output is written to `stderr` on success.